nx_parallel/algorithms/approximation/connectivity.py

```diff
@@ -5,18 +5,25 @@
 import nx_parallel as nxp
 
 __all__ = [
-    "all_pairs_node_connectivity",
+    "approximate_all_pairs_node_connectivity",
```

Comment:
If you want to allow both ways to call the functions, we could do something like
```suggestion
    "approximate_all_pairs_node_connectivity", 
    "all_pairs_node_connectivity",
```
and then below (after defining the longer named function) use
```python
    all_pairs_node_connectivity = approximate_all_pairs_node_connectivity
```

But I'm fine with this PR as is.  Whcih do you prefer?

Reply:
while generating the `get_info` function we fetch all the functions in the nx_parallel package and then extract their docstrings... so having 2 `all_pairs_node_connectivity` functions in the package(one of them being in `connectivity/connectivity`(with docstring) and the other one being in `approximation/connectivity`(without docstring)) would probably result in an incorrect mapping between functions and the `additional_docs`. So, for now, I'd prefer the current implemented way, although this way does allow the user to use nx-parallel as an independent package, so I'd like to try and find if there's a better way to generate `get_info`. But, I think that's for another PR.